core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -503,9 +503,12 @@ private[spark] object AppStatusStore {
   /**
    * Create an in-memory store for a live application.
    */
-  def createLiveStore(conf: SparkConf): AppStatusStore = {
+  def createLiveStore(
+      conf: SparkConf,
+      appStatusSource: Option[AppStatusSource] = None):
```

Comment:
nit: `: AppStatusStore`?

Reply:
appStatusSource needs to be passed here as it is created in the SparkContext, do you mean something else?

Reply:
I think the comment may be that that you can move `AppStatusStore = {` on to to same line

Reply:
Ok will do thanx.

Reply:
Yep, sorry for the late reply. :(